src_fun/parser.cpp

```diff
@@ -621,9 +697,13 @@ static bool parse_expression_leaf(Token_Stream* stream, Block_Builder* builder,
         {
             bool is_using;
             if (false) parse_using:
+            {
                 is_using = true;
+            }
             else
+            {
                 is_using = false;
+            }

```

Comment:
Perhaps cleaner:

    bool is_using = false;
    if (false) { parse_using: using = true; }

Reply:
This is is a compile error because initialization of `is_using` is skipped. I will make it tidier though.